sema/discovery/discovery.py

```
@@ -152,6 +166,35 @@ def _add_triples_from_text(self, content, mimetype, source_url):
             )
         return False
 
+    def _parse_url_content(self, resp_text: str, resp_url: str):
+
+        l_s_dict = {"links": 0, "scripts": 0}
+
+        parser = LODAwareHTMLParser()
+        parser.feed(resp_text)
+        log.info(f"found {len(parser.links)} links in the html file")
```

Ruff

Logging statement uses f-string

(G004)